ui/apps/platform/src/Containers/Dashboard/PatternFly/hooks/useRestQuery.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
```

Comment:
The hook logic for the REST request for this widget was identical to that of the Violation by policy category widget, so I pulled the logic into here until we decide as a team how best we want to handle these. The latter point is also why this is stored under `Dashbord/PF/hooks` instead of the top level hooks directory.